src/styles/foundation/border-radius.scss

```diff
@@ -1,6 +1,7 @@
 $border-radius-data: (
   base: 3px,
   large: 6px,
+  extra-large: 30px,
```

Comment:
This seems like  fairly radical jump from `3px` to `6px`, all the way up to `30px`. I doubt this value needs to be shared throughout the system, and should probably be a variable local to the `Toggle` component.